### PR TITLE
CLDR-16061 add EUR to HR from 2023-01-01

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16061

- [ ] This PR completes the ticket.

note this does NOT have an end date for HRK, i could not find that yet.

### CODE REVIEW

**Comment by srl295** [comment]
I've pushed another branch https://github.com/srl295/cldr/tree/m42/cldr-16061-croatia2 with HRK becoming obsolete Jan 1st which moves it to deprecated

File: common/supplemental/supplementalData.xml [hunk 1]
```diff
@@ -545,6 +545,7 @@ The printed version of ISO-4217:2001
             <currency iso4217="HNL" from="1926-04-03"/>
         </region>
         <region iso3166="HR">
+			<currency iso4217="EUR" from="2023-01-01"/>
```
Comment (markusicu) [inline common/supplemental/supplementalData.xml:548]: Please indent consistently. The next line uses spaces.

**Comment by DavidLRowe** [comment]
https://www.avalara.com/blog/en/europe/2022/09/croatia-to-adopt-the-euro-currency-in-2023.html
https://www.sea-help.eu/en/news-general/croatia-euro-introduction-2023/
both mention a two-week window where both HRK and EUR will be valid currencies.

File: common/supplemental/supplementalData.xml (hunk 1 above)
Comment (srl295) [reply common/supplemental/supplementalData.xml]: done

**Comment by srl295** [comment]
added an end date

**Review by pedberg-icu — APPROVED** [review]

**Comment by srl295** [comment]
@pedberg-icu feel free to merge when done if needed.. i think i addressed @markusicu  concern

**Review by markusicu — APPROVED** [review]
lgtm FWIW

**Pull request merged**